nbs/01_lhs_parsing.ipynb

```diff
@@ -34,20 +34,21 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    try:
</pre><p>look in spannerlib how i checked for assertions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>In spannerlib you checked it using pytest.raises(). Should we use it in this case, instead of just:</p>

<pre class="ql-syntax" spellcheck="false">except GraphRewriteException as e:
  assert "Unable to convert LHS: Unexpected token..."
</pre><p>Using purest.raises() would require adding pytest to the graph_rewrite library's requirements, only for this single cell.</p>

Reply:
<p>we can use assert inside the except to make sure the content of the error message is ok</p>